Updated Uniswap v2 to schema version 1.1.0

steegecs opened the pull request.

this-username-is-taken commented:
you need to resolve merge conflicts in the code

this-username-is-taken commented on subgraphs/_reference_/src/common/rewards.ts:
```diff
@@ -0,0 +1,198 @@
+//////////////////////////
+// AUTHOR: CHRIS STEEGE //
```
Can you remove this? People can find the author with `git blame`. Also version is not needed for a helper lib

steegecs replied:
Yeah - I removed this file from this pull request. I'll change it in the other.

this-username-is-taken commented:
You should remove all `.DS_Store`s in the code. I've added it to .gitignore in latest commit in master so this will automatically be ignored going forward

steegecs commented:
> you need to resolve merge conflicts in the code

All the merge conflict are resolved now.

steegecs commented:
> You should remove all `.DS_Store`s in the code. I've added it to .gitignore in latest commit in master so this will automatically be ignored going forward

Removed
